css/style.css

```diff
@@ -1,6 +1,879 @@
+  padding: 0 0 0 12px;
+  margin: 14px 0 7px 0;
+}
+
+.sneaker-item__wrap {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.sneaker-item__price {
+  font-family: 'GothaPro', sans-serif;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 13px;
+  text-align: justify;
+  color: #000;
+  margin: 1px 0 0 12px;
+}
+
+.colour-circle {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 53px;
+  align-items: center;
+  padding: 0 17px 0 0;
+}
+
+.colour-circle__item {
+  width: 7px;
+  height: 7px;
+  border-radius: 50%;
+}
+
+.colour-circle__item_active {
+  width: 9px;
+  height: 9px;
+  border-radius: 50%;
+}
+
+.colour-circle__item_grey {
+  background: #C4C4C4;
+}
+
+.colour-circle__item_black {
+  background: #0E1211;
+}
+
+.colour-circle__item_red {
+  background: #B62E3C;
+}
+
+.colour-circle__item_yellow {
+  background: #FDB431;
+}
+
+.colour-circle__item_orange {
+  background: #E29D69;
+}
+
+.colour-circle__item_brown {
+  background: #762E31;
+}
+
+.colour-circle__item_darkblue {
+  background: #41479B;
+}
+
+.colour-circle__item_blue {
+  background: #257DA9;
+}
+
+/*2 секция*/
+
+.history {
+  background: #000;
+  height: 494px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-wrap: nowrap;
+  flex:0 1 auto;
+  margin-bottom: 76px;
+}
+
+.history__video {
+  max-width: 645px;
+  padding: 4px 0 0 13px;
+}
+
+.history__video_img {
+  width: 100%;
+  height: 100%;
+}
+
+.history__wrap {
+  display: flex;
+  flex-flow: column;
+  justify-content: space-around;
+  align-items: flex-start;
+  max-width: 266px;
+  height: 190px;
+  margin: 0 25px 0 0;
+}
+
+.history__title {
+  font-family: 'GothaPro', sans-serif;
+  font-size: 22px;
+  line-height: 27px;
+  text-align: justify;
+  text-transform: uppercase;
+  color: #fff;
+  margin: 0 0 13px 0;
+}
+
+.history__text {
+  font-family: 'GothaPro', sans-serif;
+  font-size: 14px;
+  line-height: 17px;
+  text-align: left;
+  color: #fff;
+  margin: 0 0 17px 0;
+}
+
+.arrow-white_img2 {
+  padding-left: 6px;
+}
+
+/*3 секция*/
+
+.article__title {
+  font-family: 'GothaPro', sans-serif;
+  font-weight: bold;
+  font-size: 22px;
+  line-height: 21px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #000;
+}
+
+@media (max-width: 1024px){
+  .conteiner {
+    max-width: 930px;
+  }
+
+  .header {
+    height: 129px;
+  }
+
+  .header__column {
+    height: 67px;
+      flex: 0 0 61px;
+  }
+
+  .header__logo {
+    flex:0 0 61px;
+  }
+  
+  .header-menu {
+      margin: 0 116px 0 0;
+      height: 56px;
+  }
+
+  .right-menu {
+      flex: 0 0 364px;
+  }
+  
+  .search-form__input {
+    height: 17px;
+    width: 161px;
+  }
+
+  .search-form__text {
+    font-size: 12px;
+    width: 141px;
+  }
+
+  .search-form__submit {
+    padding: 0px 10px 0px 10px;
+    background: url(../image/magnifier.svg) no-repeat 20% 20%;
+  }
+
+  .right-menu__link {
+      height: 22px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/axion_ann/pullRequest?prid=5730635)